### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by Smo-RBR** [tphakala/birdnet-go#28] [General]
I just wanted to say thank you for taking over the project, rebuilding it and continuing it. The old version is not so special in many respects, especially the statistics are not really nice. But it was a huge step forward to make such software easily available. And that's why I'm all the more pleased that things are now continuing with a fresh slate. Especially because I also do some software work on a voluntary basis, I know how much time it can take. So once again, thank you very much and I hope you enjoy it.

**Comment by tphakala** [comment]
Thanks for kind words, I hope my work is beneficial for community.

**Comment by tomtom215** [comment]
Echoing what @Smo-RBR  said, but felt like leaving an additional response was worth it.

I personally loved how much joy Birdnet-Pi brought me and I have been running it for over a year now, and when I saw the author has set it to maintenance mode, I was worried the project would die and leave a community behind. Your efforts have given me hope for the future and even ideas to expand my own setup, so thank you VERY MUCH! :)
